### PR TITLE
fixed aws credentials copy in Dockerfile

### PR DESCRIPTION
**Pull request by nicmr**
AWS credentials will not be added to the image during docker bild anymore but instead have to be mounted at run time. This will adress issue #14 .
Updated travis credentials for AWS api calls. It now calls the API as a special travis user. This will adress issue #12 .

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,8 +6,8 @@ services:
...
+  - secure: Ro+5WUbz5URYG+fwsMODrdAClATnsUayDuU3IdDjS8PBY0GWYaYgUmHEODkOPPT+vehR1RA72MrhKBIUaZhyQjwlXEcUhpYlSkVH955EBfmycDqcWl0gDVfc5X1E7E1oCiu4UIZrwWlY6F4fNNVlNN3GfM+MVSozYO/HJES6eadp21khzEsOvCAuU/GwywfVa4r60VdbCekK3UgyVSfmZ2B0kk1B5uhkEl/Qsjf5z2s0GvTpPF4nR6W+BEGEXu22JvhW6hldLstEYSjSMnauN8heyFdl/6z8XRy8uZUSj9KN8u+kFi27RHayC3+7ywumywfdhNosYFRTbiPNzEzKwSn9HaCD8yJf62W1b8nMLaEKlzHGJVrOapqJ73ZQ8xZj1TeWOszf2uPT5zjhFEu10aUboZgfxQhVN9/XsIpZ3tlYEOCvoSCWJ1ZJQLvtozX2NEhIVLE1Rr7POLfmZgbgh8372tugJsNlSyRRR4ImpJCWtCW7ZEYSJVQQRtEtI1oZcRTMoEz0mtCxDyQdqIyY3+vZmE/9Rq41WHyUi/HCZ5PlEn2T308wnQDv9V76CLmgHDWBmshXf5nLuHrLOvvy4du6zNBVqsun9EX9t/zuPMu8uAUiPTH8LKx/LEsI9ShzFn8AbgDfVxr1Gm8VNDzh1MjvSZQdj3IfNh+3cMkNNeI=
```
Comment (cheld) [inline .travis.yml]: If someone forks our repo, then he will use our account? I  guess we should not commit, but just use setting in Travis

File: Dockerfile [hunk 2]
```diff
@@ -16,7 +16,7 @@ RUN apk add ca-certificates
 # Create non-root user
 RUN adduser -D runner
 # Add AWS config to user
-COPY .aws /home/runner/.aws
+COPY .aws/config /home/runner/.aws/config
```
Comment (cheld) [inline Dockerfile]: This looks suspicious

**Comment by cheld** [comment]
I think credentials should be kept centralized. Please check with Felix how credentials are handled so far

File: .travis.yml (hunk 1 above)
Comment (markushinz) [reply .travis.yml]: No one can use these credentials. These credentials are encrypted with travis' pub key so only travis is able to decrypt and use them. This is a common way to store credentials in git. But we can discuss if they should be in the repo in the first place.

Comment (nicmr) [reply .travis.yml]: Quote from Travis Documentation:
*Encrypted environment variables are not available to pull requests from forks due to the security risk of exposing such information to unknown code.*

They are only available to Travis Builds for our Repository (no forks) and only to PR's from branches, not forks.
We've also created a separate IAM User for Travis and we'd like to restrict it as much as possible, so it only has access to the APIs it actually needs to complete the tests.

File: Dockerfile (hunk 2 above)
Comment (nicmr) [reply Dockerfile]: All the config contains is region information for AWS (eu-central-1)
But we could certainly mount it at runtime as well.

**Comment by nicmr** [comment]
@cheld If we were able to adress your security concerns, could we agree on merging this branch and opening a separate issue discussing centralised credential solutions?

File: .travis.yml (hunk 1 above)
Comment (cheld) [reply .travis.yml]: Thats a new travis feature. I have not used it yet. In the past credentials just have been configured in Travis (web ui) but not commited.

**Comment by cheld** [comment]
LGTM

**Pull request merged**